src/gui/fullinspector.h

```diff
@@ -50,21 +50,28 @@ public slots:
     void slotDeselectedNode(const Node& node);
     void slotDeselectedEdge(const Edge& edge);
     void slotDelete();
-    void slotChangeAttrScope(AttributesScope nodeAttrScope);
+    void slotChangeNodeAttrsScope(AttributesScope attrScope);
+    void slotChangeEdgeAttrsScope(AttributesScope attrScope);
```

Comment:
it'd be better to take a const reference instead of a copy of the attrScope, i.e.,
``` c++
    void slotChangeNodeAttrsScope(const AttributesScope& attrScope);
    void slotChangeEdgeAttrsScope(const AttributesScope& attrScope);
```